src/vs/workbench/api/common/extHost.protocol.ts

```diff
@@ -70,7 +70,7 @@ import { CandidatePort } from 'vs/workbench/services/remote/common/remoteExplore
 import { ITextQueryBuilderOptions } from 'vs/workbench/services/search/common/queryBuilder';
```

Comment:
rm ProviderResult from everywhere except for the .d.ts and thenable is a promise